src/cmd/factory/impl/common.go

```diff
@@ -110,8 +111,10 @@ func getGCAndVerifyUser(ctx context.Context, userID string) (*connector.GraphCon
 		return nil, account.Account{}, errors.Wrap(err, "finding m365 account details")
 	}
 
-	// build a graph connector
-	gc, err := connector.NewGraphConnector(ctx, acct, connector.Users)
+	// TODO: log/print recoverable errors
+	errs := fault.New(false)
+
+	gc, err := connector.NewGraphConnector(ctx, acct, connector.Users, errs)
```

Comment:
in this and other non-cli commands/helpers we should probably check the value of `errs.Errs()` so we can surface any failures. We really want this to keep the current behavior of the code until we're sure we have all parts of corso ready for best effort. Just checking the returned `err` won't show non-catastrophic failures so it may change behavior

Reply:
Honestly what's probably easiest right now is to swap default behavior from bestEffort to failFast.  That way we catch every possible failure.  Then, when we're satisfied with the integration, we can back off to bestEffort again.